components/automate-ui/src/app/entities/users/userself.actions.ts

```diff
@@ -0,0 +1,79 @@
+import { Action } from '@ngrx/store';
```

Comment:
The userself logic was moved from the user ngrx store files because we need to register for changes to the logged-in user.